GameEngine/GameEngine/Engine/BFC/BFCItem.hpp

```diff
@@ -17,8 +18,9 @@ namespace AGE
 		inline glm::vec4 getPosition() { return _position; }
 	private:
 		BFCCullableObject *_drawable = nullptr;  // 64
-		void        *_transformationInfos; //64
 		glm::vec4   _position; // 128
+	public:
+		void        *next; //64
 
```

Comment:
Nop, I'm not sure you're right. Do the test. Members are aligned based on % 64bits